techniques/general/G220.html

```diff
@@ -95,27 +95,27 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 		
 		
 		<section id="description">
-			<h2>Description</h2>
-			<p>The objective of this technique is to provide a mechanism for finding contact details in a consistent location across pages to make it easier for users to find it. The interactive item in the web page is a link to the contact details page. The programmatic and visual location is consistent on each page in the set of web pages, when viewed within the same size viewport. Activating the link brings users to a web page with contact details such as a phone number and/or email address.</p>
+			<h2>解説</h2>
+			<p>このテクニックの目的は、ページ間で一貫した場所で連絡先情報を見つけるメカニズムを提供して、利用者が簡単に見つけられるようにすることである。ウェブページ内のインタラクティブな項目は、連絡先情報ページへのリンクである。プログラム的及び視覚的な位置は、同じサイズのビューポート内で閲覧したときに、ウェブページ一式内の各ページで一貫している。リンクをクリックすると、電話番号、電子メールアドレスなどの連絡先情報が記載されたウェブページに移動する。</p>
 			
 		</section>
 		<section id="examples">
-			<h2>Examples</h2>
+			<h2>事例</h2>
 			<section class="example" id="example-1-a-link-at-the-top-of-the-page">
-				<h3>Example 1: A link at the top of the page</h3>
-				<p>An on-line job application asks for many types of information from the user, such as their identification number, but they may have several and not know which one to enter. They may need more information from someone that can answer their question when the contextual help provided does not meet their needs. One of the first links that the user reaches when tabbing through the page is titled "Contact Us". This link is also visually in the same location on each page. Activating the link brings the user to the contact details page. The contact details page has an email address for a company representative or general information inbox which is then shared with appropriate personnel.</p>
+				<h3>事例1: ページ上部のリンク</h3>
+				<p>オンラインの求人応募では、ID 番号など、利用者に様々な情報の入力を求めるが、複数の情報を持っていて、どれを入力すればよいのかわからない場合がある。提供されるコンテキストヘルプがニーズを満たしていない場合、質問に答えてくれる人からさらに情報を得る必要があるかもしれない。利用者がページをタブで移動するとき、最初に辿り着くリンクの一つが「問い合わせ」である。このリンクは、各ページで視覚的に同じ場所に表示される。このリンクをアクティブにすると、利用者は連絡先の詳細ページに移動する。連絡先の詳細ページには、会社の担当者のメールアドレス又は一般的な情報受信トレイがあり、適切な担当者と共有される。</p>
```

Comment:
「複数の情報を持って」いるのがシステムなのかユーザーなのかよくわからない感じがしました。the user - their identification number - they may have several となっているわけですが、この their と they は「利用者」と訳してしまった方が読みやすいかもしれません。

「質問に答えてくれる人からさらに情報を得る」ですが、can なので「質問に答えられる」ですかね。

上では "contact details page" を「連絡先情報ページ」としていましたが、こちらでは「連絡先の詳細ページ」となっているので揃えたいです。

最後の文は誤訳かもしれません。たぶんこうだと思います:
an email address for [(a company representative) or {general information inbox (which is then shared with appropriate personnel)}].
直訳すると「会社の担当者、又は適切な担当者と共有される一般的な情報の受信トレイのメールアドレス」で、個人のアドレスかもしくはグループアドレスが書いてある、という話かと思います。

こんな感じでどうでしょうか?

```suggestion
				<p>オンラインの求人応募では、利用者のID 番号など、様々な情報の入力を求めるが、利用者が複数の情報を持っていて、どれを入力すればよいのかわからない場合がある。提供されたコンテキストヘルプが利用者のニーズを満たしていない場合、質問に答えられる人からさらに情報を得る必要があるかもしれない。利用者がページをタブで移動するとき、最初に辿り着くリンクの一つが「問い合わせ」と名付けられている。このリンクは、各ページで視覚的に同じ場所に表示される。このリンクをアクティブにすると、利用者は連絡先情報ページに移動する。連絡先情報ページにはメールアドレスがあり、会社の代表者に届くか、又は一般問い合わせ用の受信トレイに届き、適切な担当者と共有される。</p>
```